src/wallets/hardware/LedgerWallet.ts

```diff
@@ -73,6 +66,32 @@ export default class LedgerWallet
     this.derivationPath = path;
   }
 
+  async setAccountId(accountId: string) {
+    this.accountId = accountId;
+  }
+
+  async checkAccountId(accountId: string, publicKey: string) {
+    const state = getState();
+    const provider = new providers.JsonRpcProvider(`https://rpc.${state.options.networkId}.near.org`);
+
+    const publicKeyString = publicKey;
```

Comment:
Don't think we need this variable. We can just use `publicKey` directly in the `.query()` parameter object